packages/ember-model/tests/adapter/fixture_adapter_test.js

```diff
@@ -20,16 +20,15 @@ test("fetch loads the full FIXTURES payload when id isn't specified", function()
     
   FixtureModel.FIXTURES = data;
 
+  var done = assert.async();
   FixtureModel.fetch().then(function(records) {
-    start();
-    equal(records.get('length'), data.length, "The proper number of items should have been loaded.");
+    done();
+    assert.equal(records.get('length'), data.length, "The proper number of items should have been loaded.");
```

Comment:
Perhaps it would be worth moving the `done()` to be the last statement? (I guess the codemod didn't do that for us)

Reply:
I'm guessing it doesn't make a difference as the test will keep running until all code in the current execution context is complete, i.e. finishes after the `assert.equal` call. I'll move them to avoid any confusion though.

Reply:
I've moved all the `done()` calls to the end of their blocks